cmd/k8s-operator/manifests/proxy.yaml

```diff
@@ -2,7 +2,7 @@
 # at build time and then uses to construct Tailscale proxy pods.
 apiVersion: apps/v1
 kind: StatefulSet
-metadata:
+metadata: {}
```

Comment:
? I guess a linter did this?

Reply:
it seemed like it was missing, and i wasn't sure if yaml just allowed this or not. thought being explicit was nicer